index.js

```diff
@@ -1,38 +1,50 @@
 #!/usr/bin/env node
-const { auth, getRepositories, deleteRepository } = require('./src/github');
-const { loadConfig, saveConfig } = require('./src/config');
+const CONFIG = require('./src/config');
+const GITHUB = require('./src/github');
```

Comment:
Not a fan of all uppercase namespace thing (except for `UI`, where it makes sense)... but in the end it's a matter of taste 😛 

```javascript
const Config = require('./src/config');
const GitHub = require('./src/github');
```

Reply:
👍